assets/css/app.scss

```diff
@@ -1,6 +1,7 @@
 /* This file is for your main application css. */
 @import "custom";
 @import "~bootstrap/scss/bootstrap";
+@import "feature";
```

Comment:
I wonder if we need to import a new file or can just leverage `custom` ? 🤔

Reply:
custom from my understanding is conventionally used to store scss variables for things like colors and media query breakpoints, not actual styling rules.